src/pages/wallet/wallet.connect/index.js

```diff
@@ -6,9 +6,8 @@ import WalletConnect from '@walletconnect/client';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import _ from 'lodash';
-import Rsk3 from '@rsksmart/rsk3';
+import Web3 from 'web3';
```

Comment:
Wasn't able to test it yet. I'm trying an approach setting Wallet Connect URI on an input text.  We can enable this for dev mode only. I will do PR later